### ISSUE TITLE
svelte: add svelte-preprocess as a dependency

### ISSUE THREAD

**Issue by rondonjon** [pikapkg/create-snowpack-app#93]
PR #50 has added preprocessor support for svelte (which is great :+1:, thanks), but I struggled to get it to work, and the error message that I was getting honestly didn't help me too much:

```
  build:svelte............[ERROR] (plugin) @snowpack/plugin-svelte

▼ build:svelte

  Identifier is expected
```

I was able to solve the problem by installing `svelte-preprocess` manually.

If this should remain an optional dependency, I think it should be documented (or documented more prominently if it already is).

Since preprocessing is a pretty common thing in svelte, I would rather suggest to just add `svelte-preprocess` as a dependency directly though.

**Comment by FredKSchott** [comment]
Thanks for filing! I'm surprised to hear that, i don't see anywhere in the docs a mention that you need `svelte-preprocess` installed for `svelte.preprocess()` to run. https://svelte.dev/docs#svelte_preprocess

Mind adding some steps for me to reproduce in the Svelte CSA app? What your `svelte.config.js` file looks like and any dependencies you installed on top of the base setup it should be enough to debug. Thanks!

**Comment by rondonjon** [comment]
Hi @FredKSchott, thanks for the superfast reponse :rocket: :+1: :+1:

Preliminary note: as stated in #92, I wasn't able to run CSA directly, so the following is based on a clone of the CSA repo and the directory is `templates/app-template-svelte`.

1. Created a `svelte.config.js` as follows:

```
const sveltePreprocess = require("svelte-preprocess");

module.exports = {
  preprocess: sveltePreprocess()
};
```

2. Created a `tsconfig.json` (not sure if this matters)

```
{
  "compilerOptions": {
    "allowSyntheticDefaultImports": true,
    "baseUrl": ".",
    "esModuleInterop": true,
    "module": "commonjs",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "target": "es6"
  }
}
```

3. Added `language=` attributes to `src/App.svelte` as follows:

```
<script lang="typescript">
	const message = 'Learn Svelte';
</script>

<style lang="scss">
```

4. Then executed `yarn` and `yarn start`

-> received a pretty clear error about `node-sass,sass` being missing, so I installed these.

5. Executed `yarn start` once more and received the error above

**Comment by FredKSchott** [comment]
haha no worries! I figured this may have been the case :) 

I'm still surprised you got "Identifier is expected" and not "svelte-preprocess" not found. Tracked it down to us swallowing the error, fixed in b22b1c3
